ember_debug/object-inspector.js

```diff
@@ -207,7 +207,7 @@ export default EmberObject.extend(PortMixin, {
     if (value instanceof Error) {
       value = value.stack;
     }
-    this.get("adapter").log('Ember Inspector ($E): ', value);
+    this.get("adapter").log('Ember Inspector ($E): ', inspect(value), value);
```

Comment:
I'm thinking that although it is uncommon as there would be no reason to send it to the console, `$E` could also be a string, number, boolean, null, or undefined (for example when you `$E` a value in the object inspector that is not an object). In this case there would be no need to print the `toString()` (and possibly misleading since the number `1` would then appear as `"1"`). A more common case is when it's an `Error` and the value is the stack trace string of that error (see line 208).

So I suggest we first check if it's truthy (to avoid nulls because `typeof null === 'object'`) and if it's an object before the inspected value.